src/components/Button.js

```diff
@@ -135,30 +136,47 @@ const Button = React.createClass({
         ':hover': {
           color: StyleConstants.adjustColor(this.props.primaryColor, -8),
           fill: StyleConstants.adjustColor(this.props.primaryColor, -8),
-          transition: 'all .2s ease-in'
+          transition: 'all .2s ease-in',
+          borderColor: StyleConstants.Colors.FOG,
+          borderRadius: 2,
+          borderWidth: 1
```

Comment:
`borderWidth` can be combined with `borderColor` = `border: '1px solid' + StyleConstants.Colors.FOG`